var/spack/repos/builtin/packages/pacbio-dextractor/package.py

```diff
@@ -0,0 +1,50 @@
+##############################################################################
+# Copyright (c) 2013-2016, Lawrence Livermore National Security, LLC.
+# Produced at the Lawrence Livermore National Laboratory.
+#
+# This file is part of Spack.
+# Created by Todd Gamblin, tgamblin@llnl.gov, All rights reserved.
+# LLNL-CODE-647188
+#
+# For details, see https://github.com/llnl/spack
+# Please also see the NOTICE and LICENSE files for our notice and the LGPL.
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU Lesser General Public License (as
+# published by the Free Software Foundation) version 2.1, February 1999.
+#
+# This program is distributed in the hope that it will be useful, but
+# WITHOUT ANY WARRANTY; without even the IMPLIED WARRANTY OF
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the terms and
+# conditions of the GNU Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public
+# License along with this program; if not, write to the Free Software
+# Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA 02111-1307 USA
+##############################################################################
+from spack import *
+
+
+class PacbioDextractor(MakefilePackage):
+    """The Dextractor and Compression Command Library. This is a special
+       fork required by some pacbio utilities."""
+
+    homepage = "https://github.com/PacificBiosciences/DEXTRACTOR"
+    url      = "https://github.com/PacificBiosciences/DEXTRACTOR"
+
+    version('2016-08-09',
+            git='https://github.com/PacificBiosciences/DEXTRACTOR.git',
+            commit='89726800346d0bed15d98dcc577f4c7733aab4b1')
+
+    depends_on('hdf5~mpi')
```

Comment:
I would change it to `hdf5`. You want people to be able to build it however they want. If they don't want MPI support, they can use:
```
$ spack install pacbio-dextractor ^hdf5~mpi
```
If you use `hdf5~mpi`, it makes it physically impossible to build with `hdf5+mpi`, which is undesirable.